cli/cmd/generate_docs.go

```diff
@@ -50,7 +49,7 @@ keptn generate docs --dir=/some/directory`,
 		// check if output directory exists
 		if _, err := os.Stat(outputDir); os.IsNotExist(err) {
 			// outputDir does not exist
-			return errors.New(fmt.Sprintf("Error trying to access directory %s. Please make sure the directory exists.", outputDir))
+			return fmt.Errorf("Error trying to access directory %s. Please make sure the directory exists", outputDir)
```

Comment:
test case failing here, it should reflect it changes to its test file as well. e.g. cli/cmd/generate_docs_test.go